src/postgres/utils/sql.ts

```diff
@@ -153,22 +164,20 @@ namespace sql {
   }
 
   /**
-   * Escapes a Sql identifier. Adapted from the `pg` module.
+   * Escapes a Sql identifier. From the `pg` module.
    *
    * @private
    * @see https://github.com/brianc/node-postgres/blob/a536afb1a8baa6d584bd460e7c1286d75bb36fe3/lib/client.js#L255-L272
    */
   function escapeSqlIdentifier (str: string): string {
-    let escaped = '"'
-
-    for (const c of str) {
-      if (c === '"') escaped += c + c
-      else escaped += c
-    }
-
-    escaped += '"'
+    return Client.prototype.escapeIdentifier(str)
```

Comment:
I'm not sure; it could be broken in a point release. I'm hoping if it is then the `require()` will fail making it very obvious though.